CHANGELOG.md

```diff
@@ -23,6 +23,10 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 - Update Erlang/OTP to 26.1.
 - Update container base image to Debian `Bookworm`.
 - Container user has changed from `root` to `nobody`.
+- incoming_introspection events display introspection as a map rather than
```

Comment:
Nit: this needs a `BREAKING` notice since by default it's breaking (you need to take an action for it not being breaking)